### ISSUE TITLE
content: seed how models are actually trained, and fix a third bent edge

### ISSUE THREAD

**Issue by opendroid** [opendroid/the-infinity#296]
Fourth pass with #288's lens. The gap this time is the training recipe itself — the parts of it that are in every implementation and named nowhere:

```
teacher-forcing   0 mentions      exposure-bias      0 mentions
label-smoothing   0 mentions      self-play          0 mentions
receptive-field   5 mentions, never defined
transfer-learning 1 mention       sequence-to-sequence  0
ensemble-methods  absent
```

**`teacher-forcing` and `exposure-bias` are the pair that matters most.** Every autoregressive model in this corpus is trained on ground-truth prefixes and used on its own outputs, and the corpus does not have a word for that mismatch — despite `autoregressive-decoding`, `beam-search-tradeoffs`, `hallucination` and `reasoning-trace` all being downstream of it.

**`ensemble-methods` is a third bent edge**, after `active-learning` and `bayesian-inference` in #294. Seeding `deep-ensemble` I drafted `requires: ensemble-methods`, found it absent, and settled for `bias-variance-tradeoff`. The variance argument is real but it is not the prerequisite a reader needs.

`receptive-field` is the other kind of gap: mentioned in five bodies and defined in none, and both `convolutional-network` and `dilated-convolution` lean on it.

## Scope — eight nodes

`teacher-forcing` · `exposure-bias` · `label-smoothing` · `transfer-learning` · `sequence-to-sequence` · `self-play` · `receptive-field` · `ensemble-methods`

Scoping constraints:

- `transfer-learning` must not restate `fine-tuning`, which is one method for it. The subject is what transfers and what does not — early layers against late ones — and why that is a property of the representation rather than of the procedure.
- `sequence-to-sequence` must not restate `encoder-decoder`. That node is the architecture; this is the framing that made it, and the bottleneck that motivated attention.
- `ensemble-methods` should be the prerequisite `deep-ensemble` wanted, and that edge should be repointed in the same PR.

## Acceptance criteria

- [ ] Eight nodes, schema-valid, connected — no restatement of `fine-tuning`, `encoder-decoder` or `deep-ensemble`
- [ ] `deep-ensemble` repointed at `ensemble-methods`
- [ ] Every arXiv id verified by title; pre-arXiv primary sources in `origin`
- [ ] Figures assigned from arithmetic; captions name their control
- [ ] `make golden` regenerated; all gates green
- [ ] Verified in a second pass that re-derives each math body
- [ ] Teal lobe moved when the verification empties it

Refs #61, #288, #294
